src/components/header/Header.jsx

```diff
@@ -0,0 +1,93 @@
+import { useState, useEffect } from 'react';
+const Header = () => {
+    const [isNavOpen, setIsNavOpen] = useState(false);
+
+    const toggleNav = () => {
+        setIsNavOpen(!isNavOpen);
+    };
+
+    useEffect(() => {
+        const checkScreenSize = () => {
+            if (window.innerWidth >= 1024) {
+                setIsNavOpen(false);
+            }
+        };
+
+        window.addEventListener('resize', checkScreenSize);
+
+        checkScreenSize();
+
+        return () => window.removeEventListener('resize', checkScreenSize);
+    }, []);
+    return (
+        <header className="px-4 opacity-100 backdrop-blur-18 relative ">
+            <div className="container mx-auto p-6 flex justify-between items-center">
+                <a href="/" className="flex gap-4 items-center">
+                    <div className="w-[17px] h-6 md:w-6 md:h-8 relative z-10">
+                        <img className="w-full h-full" src="./logo__1.png" alt="Logo 1" />
```

Comment:
Please remove __ from the name instead use - @itsimran02. Also, rename logo to `brand-logo`.